packages/data/tests/unit/adapters/dimensions/elide-test.ts

```diff
@@ -0,0 +1,269 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import config from 'ember-get-config';
+import { asyncFactsMutationStr } from 'navi-data/gql/mutations/async-facts';
+import { asyncFactsQueryStr } from 'navi-data/gql/queries/async-facts';
+import ElideDimensionAdapter from 'navi-data/adapters/dimensions/elide';
+import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
+import DimensionMetadataModel from 'navi-data/models/metadata/dimension';
+import { Response } from 'miragejs';
+// @ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { TestContext } from 'ember-test-helpers';
+
+interface MirageTestContext extends TestContext {
+  server: TODO;
+}
+
+type ServerRequest = {
+  requestBody: string;
+};
+
+module('Unit | Adapter | Dimensions | Elide', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  test('find', async function(this: MirageTestContext, assert) {
+    assert.expect(6);
+    const {
+      navi: { dataSources }
+    } = config;
+    const adapter: ElideDimensionAdapter = this.owner.lookup('adapter:dimensions/elide');
+    const TestDimensionColumn: DimensionColumn = {
+      columnMetadata: DimensionMetadataModel.create({
+        id: 'dimension1',
+        source: 'elideTwo',
+        tableId: 'table1'
+      }),
+      parameters: {
+        foo: 'bar'
+      }
+    };
+    const HOST = dataSources.find(d => d.name === 'elideTwo') || dataSources[0];
+
+    let callCount = 0;
+    let queryVariable: string;
+    let queryId: string;
+
+    let response: TODO;
+    this.server.post(`${HOST.uri}/graphql`, function(_: TODO, { requestBody }: ServerRequest) {
+      callCount++;
```

Comment:
Since we are using the fact adapter, do we need to test this again? Can we use ember-cli-mirage-graphql?

Reply:
I'm working on testing the dimension service by hitting the `ember-cli-mirage-graphql` stuff in a separate PR.